falcon/testing/client.py

```diff
@@ -18,13 +18,13 @@
 WSGI callable, without having to stand up a WSGI server.
 """
 
-import json  # Don't use util.json so we can sanity-test compatibility with ujson output
 import platform
 import re
 import wsgiref.validate
 
 from six.moves import http_cookies
 
+from falcon import util
 from falcon.testing import helpers
 from falcon.testing.srmock import StartResponseMock
 from falcon.util import CaseInsensitiveDict, http_date_to_dt, to_query_str
```

Comment:
I think it would be more consistent with the existing code to import ``json`` here and use it directly, rather than ``from falcon import util``.

Reply:
The problem is that the parameter is also called ``json`` which would shadow or be shadowed by the import name.

I could alias that to, for instance, ``util_json``, or do like testing/request.py``, I have tried something in that vein (see my first commit in the PR), but it got sort of ugly to  to support both ``json`` and ``ujson``.

How about using just ``json``:

```
@@ -18,13 +18,14 @@ This package includes utilities for simulating HTTP requests against a
 WSGI callable, without having to stand up a WSGI server.
 """
 
+from json import dumps as json_dumps, loads as json_loads
 import platform
 import re
 import wsgiref.validate
```